consensus/istanbul/backend/engine.go

```diff
@@ -426,6 +440,20 @@ func (sb *backend) Prepare(chain consensus.ChainReader, header *types.Header) er
 		logger.Info("Put voteData", "num", number, "data", hex.EncodeToString(header.Vote))
 	}
 
+	if chain.Config().IsRandaoForkEnabled(header.Number) {
+		prevMixHash := parent.MixHash
+		if chain.Config().IsRandaoForkBlockParent(parent.Number) {
+			prevMixHash = params.ZeroMixHash
+		}
```

Comment:
How about making a wrap for this getter? https://github.com/klaytn/klaytn/pull/2014/files#diff-44abfea3bdbe16ce301a955732b485d1f0af2cc6c0c4cee3fbce0feb6ce051b0R257 this can be shorter as well

Reply:
Do you mean something like this? I'll add in #2030.

```go
func getMixHash(header, chain) {
    if chain.Config().IsRandaoForkBlockParent(header.Number) {
        return params.ZeroMixHash
    } else {
        return params.MixHash
    }
}
```

Reply:
Yes. I'll take a look. thanks

Reply:
Fixed 75d0c836.